style.css

```diff
@@ -96,3 +96,272 @@ a {
+  margin-right: 40px;
+  padding-right: 24px;
+}
+
+.tags {
+  width: 297px;
+  height: 40px;
+  gap: 12px;
+}
+
+.tags img {
+  width: 49px;
+  height: 40px;
+  gap: 30px;
+  flex-direction: row;
+  padding-top: 20px;
+  padding-bottom: 18px;
+}
+
+.tags button {
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 3%;
+  color: #fff;
+}
+
+.frame43 {
+  width: 375px;
+  height: 2436px;
+  background-color: #fff;
+}
+
+.mask-group-one {
+  width: 327px;
+  height: 110px;
+  background-image: url("image/imgPlaceholderTwo.png");
+  background-repeat: no-repeat;
+  margin-left: 24px;
+  padding-bottom: 386px;
+  background-position-x: right;
+  background-position-y: 120px;
+}
+
+.mask-group-one h1 {
+  padding-top: 200px;
+  padding-left: 16px;
+  padding-right: 16px;
+  width: 295px;
+  height: 81px;
+  color: #fff;
+  font-family: Crete Round, sans-serif;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: left;
+  vertical-align: top;
+  margin-bottom: 5px;
+}
+
+.mask-group-one p {
+  margin-left: 24px;
+  margin-right: 24px;
+  padding-left: 16px;
+  padding-right: 16px;
+  margin-bottom: 0;
+  width: 295px;
+  height: 81px;
+  color: #fff;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  line-height: 124%;
+  text-align: left;
+  vertical-align: top;
+}
+
+.mask-group-one button {
+  width: 85px;
+  height: 40px;
+  background-color: #a9a9a9;
+  color: #fff;
+  gap: 30px;
+  flex-direction: row;
+  font-family: Inter, sans-serif;
+  font-size: 18px;
+  line-height: 20px;
+  line-height: 110%;
+  text-align: left;
+  vertical-align: top;
+}
+
+.bottom {
+  width: 100%;
+  height: 80px;
+  margin-top: 15px;
+  background-color: #ff6b00;
+}
+
+.bottom p {
+  color: #fff;
+  padding-top: 20px;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  line-height: 117%;
+  text-align: center;
+  letter-spacing: 3%;
+  margin: 12px;
+}
+
+.rectangle {
+  width: 375px;
+  height: 1400px;
+  margin-top: 120px;
+  background-color: #1c1a19;
+  margin-left: 24px;
+  margin-bottom: 0;
+}
+
+.rectangle h1 {
+  padding-top: 120px;
+  padding-left: 24px;
+  padding-right: 24px;
+  color: #ff6b00;
+  font-family: Crete Round, sans-serif;
+  font-size: 52px;
+  line-height: 88px;
+  line-height: 96%;
+  text-align: left;
+  vertical-align: top;
+  letter-spacing: 0.37px;
+}
+
+.rectangle p {
+  padding-left: 0;
+  font-family: Inter, sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  line-height: 124%;
+  text-align: left;
+  vertical-align: top;
+}
+
+.rectangle button {
+  width: 161px;
+  height: 48px;
+  margin-top: 50px;
+  margin-left: 24px;
+  margin-right: 190px;
+  background-color: #ff6b00;
+  font-family: Inter, sans-serif;
+  color: #fff;
+  font-size: 17px;
+  line-height: 24px;
+  line-height: 117%;
+  text-align: center;
+  letter-spacing: 3%;
+}
```

Comment:
- [ ] It would be nice you remove the default border styles on all your buttons on the page as they are currently making your design not look like the Figma design.
- [ ] Please apply interactions on the button styling: Active, Hover, and Pressed.

** Your design **
![image](https://user-images.githubusercontent.com/50941074/160246672-3f5757a6-1788-4164-b369-87db3706a5e8.png)

** The Figma Design **
![image](https://user-images.githubusercontent.com/50941074/160246689-98869e18-07fd-41d1-bf56-01a148d2031f.png)

Hint: You can apply this `border: none;` style on all your buttons. 👍

Reply:
not resolved